src/marqo/tensor_search/tensor_search.py

```diff
@@ -1770,7 +1886,8 @@ def get_content_vector(possible_jobs: List[VectorisedJobPointer], job_to_vectors
 
     Raises runtime error if is not found
     """
-    content_type = 'image' if treat_urls_as_images and _is_image(content) else 'text'
+    content_type = 'text' if infer_modality(content) == Modality.TEXT else 'media'
```

Comment:
Do we ever get `media` from `jobs[vec_job_pointer.job_hash].content_type`?

Reply:
yes, for when the user uses a video audio or image link in the query